pkgs/development/compilers/flutter/patches/move-cache.patch

```diff
@@ -43,16 +43,16 @@ index 11e3bf3e11..39d6fae0d1 100644
 +    }
 +
      final File lockFile =
--      _fileSystem.file(_fileSystem.path.join(flutterRoot, 'bin', 'cache', 'lockfile'));
+-      _fileSystem.file(_fileSystem.path.join(flutterRoot!, 'bin', 'cache', 'lockfile'));
 +      _fileSystem.file(_fileSystem.path.join(globals.fsUtils.homeDirPath, '.cache', 'flutter', 'lockfile'));
```

Comment:
I think this  `globals` import is missing

Reply:
I'll build and test the binaries before I mark the PR ready for review, I think I'd probably notice any mistakes then, right?